package.json

```diff
@@ -3,10 +3,16 @@
   "description": "Jsonnet libraries to help with GoCD",
   "type": "module",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "eslint": "eslint --config './.eslintrc.json' .",
+    "prettier": "prettier .",
+    "lint": "npm run eslint && npm run prettier -- --check",
+    "lint:fix": "npm run eslint -- --fix && npm run prettier -- -w"
   },
   "devDependencies": {
-    "ava": "^5.3.0"
+    "ava": "^5.3.0",
+    "eslint": "^8.53.0",
+    "prettier": "^3.1.0"
```

Comment:
I want prettier to continue working even when this repo stops using js (we'll convert to python someday soon-ish). Please use pre-commit for these responsibilities. For example:

https://github.com/getsentry/sentry/blob/07f4f07b9/.pre-commit-config.yaml#L114

P2: I'd rather not have to revisit this in future, but you can punt if you decide it's too annoying.

Reply:
punting